contribute/documentation/README.md

```diff
@@ -0,0 +1,235 @@
+
+GitHub captures your request as an **Issue** logged against the repository.
+
+Before you begin:
+
+- Create a GitHub account.
+
+To request a change, complete the following steps:
+
+1.  While viewing the topic, click **Request a change**.
+
+The Issue title auto-populates with the location of the file about which you are requesting a change.
+
+![Request a change](request-change.png)
+
+2. Enter a change request description.
+
+3. Add the **type/docs** label.
+
+4. Click **Submit new issue**.
+
+### Edit a topic
+
+If you want to recommend a small change, such as suggesting a correction to a topic, you can edit the topic directly in GitHub. You are not required to fork and clone the repo to use this approach.
+
+Other small changes might include:
+
+- Adding steps to a task
+- Adding clarifying language to a concept
+- Providing an example
+
+Before you begin:
+
+- Create a GitHub account.
+
+To edit a topic, complete the following steps:
+
+1. While viewing the topic you want to edit, click **Edit this page**.
+
+![Edit a topic](edit-file.png)
+
+2. Add your changes to the topic.
+
+3. Scroll to the bottom of the page and enter a branch name.
+
+   For example, enter `clarified dashboard panel definition`.
+
+4. Click **Commit**.
+
+   GitHub prompts you to create a PR.
+
+5. Complete the prompts provided in the body of the PR.
+
+6. Click **Create pull request**.
+
+### Write a topic
+
+At Grafana Labs, we use the principles of topic-based authoring when we write technical documentation. Topic-based authoring provides guidelines for writing three _types_ of technical documentation: concept, task, and reference. Before you begin writing, establish the topic type you want to write.
+
+#### Understanding topic types
+
+Technical content is divided into three topic types: concept, task, and reference.
+
+- **Concept**: A concept topic explains _what_ a feature (or idea) is, and why it is important.
+- **Task**: A task topic explains _how_ to complete an end user procedure in the system. Task topics contain steps.
+- **Reference** A reference topic contains lookup information that a user might consult when they complete a task. Documenting a list of values with descriptions is a common form of reference topic.
+
+**Example**
+
+Suppose you are writing content for a site called _Doggie handbook_. You might organize your topics like this:
+
+**Concepts**
+
+- What a dog is
+- Brief history of dogs
+- Why you might want a dog
+- Tasks dogs can be trained to do
+
+**Tasks**
+
+- Feed the dog
+- Groom the dog
+- Train the dog
+
+**References**
+
+- List of dog equipment you will need
+- Table of breeds that includes breed name, size range, short or long hair, and type of dog
+
+#### Prepare your environment
+
+Before you begin writing, we recommend that you fork and clone the Grafana repository so that you can use a text editor locally to create branches, commit your changes, and create a PR.
```

Comment:
fork and clone, or do we just want folks to clone?

Reply:
@achatterjee-grafana, do folks need to fork from Grafana to make a change rather than just clone?

Reply:
External users will fork, not clone the repo.